src/renderer/entities/wallet/model/__tests__/wallet-model.test.ts

```diff
@@ -100,4 +100,31 @@ describe('entities/wallet/model/wallet-model', () => {
     expect(scope.getState(walletModel.$wallets)).toEqual(wallets.concat({ ...newWallet, isActive: true }));
     expect(scope.getState(walletModel.$accounts)).toEqual(walletMock.accounts.concat(newAccounts));
   });
+
+  test('should update $wallets, $accounts on removeWallet', async () => {
+    const wallets = walletMock.getWallets(0);
+    const [removedWallet, ...newWallets] = wallets;
+
+    const removedAccounts = walletMock.accounts.filter((a) => a.walletId === removedWallet.id);
+
+    jest.spyOn(storageService.wallets, 'delete').mockResolvedValue();
+    const deleteAccountsSpy = jest.spyOn(storageService.accounts, 'deleteAll').mockResolvedValue();
+
+    const scope = fork({
+      values: new Map().set(walletModel.$wallets, wallets).set(walletModel.$accounts, walletMock.accounts),
+    });
+
+    expect(scope.getState(walletModel.$wallets)).toHaveLength(wallets.length);
+
+    await allSettled(walletModel.events.walletRemoved, {
+      scope,
+      params: removedWallet,
+    });
+
+    expect(deleteAccountsSpy).toHaveBeenCalledWith(removedAccounts.map((a) => a.id));
```

Comment:
It's ok to check service function call, but do we actually care about it?
It's more important to get the right `$wallets` after deletion completes